### PR TITLE
Added documentation for nina

### PR DESCRIPTION
**Pull request by DeerMaximum**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/56647
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2842
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/nina.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ha_codeowners:
+  - '@DeerMaximum'
+ha_domain: nina
+ha_platforms:
+  - binary_sensor
+---
+
+The [`nina`](https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/warn-app-nina_node.html) integration displays warnings from the [`Bundesamtes für Bevölkerungsschutz und Katastrophenhilfe`](https://www.bbk.bund.de/) in Germany.
+
+For each county/city its create warning slots which change to Unsafe when warnings are present. The text of the warning and the metadata are stored in the attributes of the slots.
```
Comment (marvin-w) [inline source/_integrations/nina.markdown]:
```suggestion
For each county/city it creates warning slots which change to Unsafe when warnings are present. The text of the warning and the metadata are stored in the attributes of the slots.
```

**Review by marvin-w — DISMISSED** [review]
Looks good! Can be merged as soon as the parent PR is ready.

File: source/_integrations/nina.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+title: NINA
+description: Instructions on how to set up NINA warnings in Home Assistant.
+ha_category:
+  - Binary Sensor
+ha_release: 2021.11.0
```
Comment (frenck) [inline source/_integrations/nina.markdown]:
```suggestion
ha_release: 2021.2
```

File: source/_integrations/nina.markdown [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@DeerMaximum'
+ha_domain: nina
+ha_platforms:
+  - binary_sensor
+---
+
+The [`nina`](https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/warn-app-nina_node.html) integration displays warnings from the [`Bundesamtes für Bevölkerungsschutz und Katastrophenhilfe`](https://www.bbk.bund.de/) in Germany.
```
Comment (frenck) [inline source/_integrations/nina.markdown]:
```suggestion
The [NINA](https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/warn-app-nina_node.html) integration displays warnings from the [Bundesamtes für Bevölkerungsschutz und Katastrophenhilfe](https://www.bbk.bund.de/) in Germany.
```

File: source/_integrations/nina.markdown [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+For each county/city it creates warning slots which change to Unsafe when warnings are present. The text of the warning and the metadata are stored in the attributes of the slots.
```
Comment (frenck) [inline source/_integrations/nina.markdown]:
```suggestion
For each county/city it creates warning slots that change to Unsafe when warnings are present. The text of the warning and the metadata are stored in the attributes of the slots.
```

**Comment by marvin-w** [comment]
@DeerMaximum Can you please merge next once again into this branch to retrigger CI?

File: source/_integrations/nina.markdown (hunk 2 above)
Comment (marvin-w) [inline source/_integrations/nina.markdown]:
```suggestion
ha_release: 2022.1.0
```

**Review by frenck — DISMISSED** [review]

File: source/_integrations/nina.markdown (hunk 2 above)
Comment (frenck) [reply source/_integrations/nina.markdown]: This suggestion is incorrect. We don't prefix the minor part of releases with a leading `0`, additional, there will not be a `2022.1` release, as the first upcoming release will be `2022.2`.

**Comment by frenck** [comment]
> @DeerMaximum Can you please merge next once again into this branch to retrigger CI?

Please don't merge as suggested by @marvin-w, instead, rebase. For more information see our developer documentation:

https://developers.home-assistant.io/docs/development_catching_up

File: source/_integrations/nina.markdown (hunk 2 above)
Comment (marvin-w) [reply source/_integrations/nina.markdown]: Lets just use your suggestion then :+1:

File: source/_integrations/nina.markdown (hunk 3 above)
Comment (marvin-w) [inline source/_integrations/nina.markdown]:
```suggestion
The [NINA](https://www.bbk.bund.de/DE/Warnung-Vorsorge/Warn-App-NINA/warn-app-nina_node.html) integration displays warnings from the [Bundesamt für Bevölkerungsschutz und Katastrophenhilfe](https://www.bbk.bund.de/) in Germany.
```

File: source/_integrations/nina.markdown (hunk 2 above)
Comment (marvin-w) [reply source/_integrations/nina.markdown]: This should be 

```suggestion
ha_release: 2022.2
```

**Review by marvin-w — APPROVED** [review]
Thanks @DeerMaximum!

**Review by klaasnicolaas — APPROVED** [review]
LGTM ✅

**Pull request merged**